backend/src/configs.clj

```diff
@@ -0,0 +1,27 @@
+(ns configs)
+
+(def out-of-turn "Out of turn play")
+
+(def no-opp "Waiting for an opponent")
+
+(def play "Playing")
+
+(def wait "Waiting for opponent's play")
+
+(def too-players "Too many players")
+
+(defn ini-hand
```

Comment:
WOAH! You made a random rules change, can you please make this a vector of `{:power 10}`. Although it `should` be enough to make the change, I would rather have it isolated in another issue. Also, is there a test for this?

In any case, please make this be {:power 10} for now.

Reply:
The issue was to include all those changes on configs.clj.
Being able to change the file while passing the test is what I understood as the test itself. I'll change it, but I needed to check I was able to change it without any problem.